vllm/config/compilation.py

```diff
@@ -210,7 +210,11 @@ class CompilationConfig:
 
     # Inductor capture
     use_inductor: bool = True
```

Comment:
```suggestion
    use_inductor: Optional[bool] = None
```
This way we can detect if specified by the user and warn